meta.yaml

```diff
@@ -16,39 +16,50 @@ requirements:
   host:
     - gensim >=3.4.0
     - matplotlib >=2.2
-    - networkx >=2.2,<2.4
+    - networkx >=2.2
     - numpy >=1.14
     - pandas >=0.24
     - pip
-    - python
+    - python >=3.6
     - scikit-learn >=0.20
     - scipy >=1.1.0
-    - tensorflow 1.14.*
+    - tensorflow >=2.1.0
     - ipython
     - ipykernel
   run:
     - gensim >=3.4.0
     - matplotlib >=2.2
-    - networkx >=2.2,<2.4
+    - networkx >=2.2
     - numpy >=1.14
     - pandas >=0.24
-    - python
+    - python >=3.6
     - scikit-learn >=0.20
     - scipy >=1.1.0
-    - tensorflow 1.14.*
+    - tensorflow >=2.1.0
     - ipython
     - ipykernel
 
 test:
   imports:
     - stellargraph
+    - stellargraph.connector
+    - stellargraph.connector.neo4j
     - stellargraph.core
     - stellargraph.data
+    - stellargraph.datasets
+    - stellargraph.interpretability
+    - stellargraph.interpretability.saliency_maps
     - stellargraph.layer
     - stellargraph.mapper
     - stellargraph.utils
-    - stellargraph.utils.saliency_maps
-
+  requires:
+    - pytest
+    - pytest-benchmark >= 3.1
+  source_files:
+    - tests
+  commands:
+    - pip install rdflib
```

Comment:
Why this here? Could it be part of the `requires`? If not, maybe a comment.

Reply:
rdflib is here as there's no Conda package, however it's only used in one test - running a subset of tests allows us to remove it.